contrib/building.md

```diff
@@ -197,35 +197,60 @@ Other editors should also be able to be configured to use the `compile_commands.
 
 ## Building Realm JS
 
+In most cases, it's not required to build the SDK explicitly. You can either simply download the package from `npm` or run one of the test scripts in either of the `integration-tests/environments` (which will drive the dependent build-scripts automatically). If you want to invoke these scripts manually, see the individual sections below:
+
+### Building the SDK
+
+Most of Realm JS is platform independent code (commonly referred to as the SDK), which is built explicitly by running:
+
+```sh
+npm run build:ts --workspace realm
+```
+
 ### Building for iOS
 
 You can build and bundle for iOS by running the following command from the root directory:
 
 ```sh
-npm run build:ios --workspace realm
-npm run bundle --workspace realm
+# Pre-build Realm Core into an XCFramework
+npm run prebuild-apple --workspace realm
```

Comment:
Super nitpicky, but curious why we use a `-` for one separator and `:` for the other.

Reply:
The honest reason is because I don't want `prebuild-node` to act as an NPM pre-script for the existing `build:node` and it would if it was `prebuild:node` 🙈 We could revisit this later, but this was a fast and (only slightly dirty workaround).